src/kvstore/RocksEngine.cpp

```diff
@@ -234,7 +233,6 @@ RocksEngine::rangeWithPrefix(const std::string& start,
                              const std::string& prefix,
                              std::unique_ptr<KVIterator>* storageIter) {
     rocksdb::ReadOptions options;
-    options.prefix_same_as_start = true;
```

Comment:
Good job，thx
Perhaps just removing `prefix_same_as_start` is enough. In this way rocksdb uses the default value of `prefix_same_as_start`.